gossip/evmstore/carmen.go

```diff
@@ -73,6 +73,27 @@ func (c *CarmenStateDB) GetLogs(txHash common.Hash, blockHash common.Hash) []*ty
 	return logs
 }
 
+func (c *CarmenStateDB) Logs() []*types.Log {
```

Comment:
Is there any test that would verify that we produce correct logs and/or this conversion is correct?

If not, we should add this to the Todo list.

Reply:
yes, logs are checked here for all tests: https://github.com/Fantom-foundation/go-ethereum-sonic/blob/eth1.14.8/tests/state_test_util.go#L229

(this is the reason I added this method)